src/containers/Tenant/Diagnostics/Consumers/Consumers.scss

```diff
@@ -3,4 +3,11 @@
         width: 200px;
         margin-bottom: 20px;
     }
+    &__loader {
```

Comment:
I see this styles in many places where uikit loader is used. May be it would be reasonable to create an internal Loader component and use it throughout the app?